### PR TITLE
[Doc] Restructure Translation Documentation

### PR DESCRIPTION
**Pull request by djhi**
Far from perfect yet but that's a start

### CODE REVIEW

**Review by slax57 — COMMENTED** [review]
Otherwise, this looks very fine to me, gj! 🙂

File: docs/navigation.html
Comment (slax57) [reply docs/navigation.html]: You still miss the link to 'TranslationSetup.md' if I'm not mistaken

**Comment by slax57** [comment]
I'd like to wait for @fzaninotto 's approval before merging this one 🙂

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**
